src/relation_engine_server/server.py

```diff
@@ -67,20 +66,28 @@ def api_call(path):
     return _json_resp(result, 200)
 
 
-def _get_version(version_str):
-    """From a list of path parts, initialize and validate a version int for the api."""
+def _get_version_and_path(path_parts):
+    """
+    From a list of path parts, initialize and validate a version int for the api.
+    Returns pair of (version_int, path_str)
+    """
+    version_str = path_parts[0]
     max_version = len(_API_VERSIONS)
     # Make sure the version looks like 'v12'
     if not re.match(r'^v\d+$', version_str):
-        raise InvalidParameters('Make a request with the format /api/<version>/<path...>')
+        # Fallback to v1 for paths like /api/<path...> with no version option
+        # TODO temporary
+        return (1, '/'.join(path_parts))
+        # raise InvalidParameters('Make a request with the format /api/<version>/<path...>')
```

Comment:
I think this is OK for queries (and thus for our needs) but we should bear in mind you did add one breaking change to V1 which was to change GET \update_specs to PUT \specs

Reply:
I'm pretty positive none of our frontends or other dependents use those endpoints, so should be okay. We'll remove the version-less endpoints as soon as all the dependents get updated, after updating the API